### DISCUSSION TITLE
Add some extra info to the documentation

### DISCUSSION THREAD

**Discussion by rodyvansambeek** [adamralph/minver#873] [Ideas]
I recently found out about this package, and I absolutely love it. However when starting out using it, I struggled with grasping the basics to use it in the indented way in production. Especially when you have background knowledge of other versioning mechanisms and are used to different approaches. The following posts all have the same similar difficulties: 

https://github.com/adamralph/minver/discussions/661
https://github.com/adamralph/minver/discussions/862
https://github.com/adamralph/minver/discussions/747
https://github.com/adamralph/minver/discussions/703

I think this can be prevented by adding some extra information in the documentation. 
I actually wrote a small blogpost about MinVer and addressed some of my issues in this specific part: 
https://www.rodyvansambeek.com/blog/simplifying-nuget-versioning-with-minver#using-minver-in-cicd-pipelines

You are welcome to copy-paste parts of the post in the docs, or please comment on any misconceptions I have in the blog post. 
I am also willing to provide a pull-request with some changes in the docs if that's easier. Let me know.

**Comment by adamralph** [comment]
> I am also willing to provide a pull-request with some changes in the docs if that's easier. Let me know.

That would be great. Thanks!

**Reply by rodyvansambeek** [reply]
No problem, will do this later this week!
